### DISCUSSION TITLE
添加测速功能

### DISCUSSION THREAD

**Discussion by robin12jbj** [2dust/v2rayNG#1467] [Ideas]
希望添加测速功能

**Comment by KmooooU** [comment]
有的

**Reply by guanjian1212** [reply]
v1.8.0，点击右上角三个点，没有测速功能，只有：
服务重启
删除全部配置
删除无效配置（先测试）
导出全部（非自定义）配置至剪贴板
测试全部配置Tcping
测试全部配置真连接
按测试结果排序
更新订阅
